build.gradle.kts

```diff
@@ -24,6 +24,8 @@ if (JavaVersion.current() != JavaVersion.VERSION_11 && JavaVersion.current() !=
   )
 }
 
+IntellijVersions.overrideBuildTarget = project.properties["overrideBuildTarget"] as String?
```

Comment:
Now that is the way of setting the property that solves the gradle cache problem. It did not make it into the previous commit 😄.